blueocean-web/src/main/js/init.jsx

```diff
@@ -43,6 +43,11 @@ exports.initialize = function (oncomplete) {
     const Extensions = require('@jenkins-cd/js-extensions');
     jenkinsMods.export('jenkins-cd', 'js-extensions', Extensions);
 
+    // Create and export a shared instance of the core
+    // js module
+    const corejs = require('@jenkins-cd/blueocean-core-js');
+    jenkinsMods.export('jenkins-cd', 'blueocean-core-js', corejs);
+
```

Comment:
Done differently now. Web bundle exports now done [on the `bundle` instance in the gulpfile](https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-web/gulpfile.js#L39-L43).

So, add `export("@jenkins-cd/blueocean-core-js")` there and `import("@jenkins-cd/blueocean-core-js@any")` wherever it's used. We could also [add a "default" `import` in js-extensions](https://github.com/jenkinsci/blueocean-plugin/blob/master/js-extensions/%40jenkins-cd/js-builder.js#L47-L54).